sunpy/instr/tests/test_aia.py

```diff
@@ -16,6 +16,8 @@
 def test_aiaprep():
     # Test that header info for the map has been correctly updated
     # Check all of these for Map attributes and .meta values?
+    # Check array shape
+    assert prep_map.data.shape == original.data.shape
```

Comment:
All AIA images are 4096x4096.  There's a Quality keyword that, if non-zero, indicates through bit flags if it's a calibration image, dark image, etc.